community/dbms/src/main/java/org/neo4j/commandline/dbms/ImportCommand.java

```diff
@@ -98,7 +103,28 @@ public String usage()
             .withArgument( new OptionalBooleanArg( "ignore-duplicate-nodes", false,
                     "If duplicate nodes should be ignored during the import." ) )
             .withArgument( new OptionalBooleanArg( "ignore-missing-nodes", false,
-                    "If relationships referring to missing nodes should be ignored during the import." ) );
+                    "If relationships referring to missing nodes should be ignored during the import." ) )
+            .withArgument( new OptionalBooleanArg( "multiline-fields",
+                    DEFAULT.multilineFields(),
+                    "Whether or not fields from input source can span multiple lines," +
+                            " i.e. contain newline characters." ) )
+            .withArgument( new OptionalNamedArg( "delimiter",
+                    allowedDelimiterCharacters,
+                    String.valueOf( COMMAS.delimiter() ), "Delimiter character between values in CSV data." ) )
+            .withArgument( new OptionalNamedArg( "array-delimiter",
+                    allowedDelimiterCharacters,
+                    String.valueOf( COMMAS.arrayDelimiter() ),
+                    "Delimiter character between array elements within a value in CSV data." ) )
+            .withArgument( new OptionalNamedArg( "max-memory",
+                    "max-memory-that-importer-can-use",
+                    String.valueOf( DEFAULT_MAX_MEMORY_PERCENT ),
+                    "Maximum memory that importer can use for various data structures and caching " +
+                            "to improve performance. If left as unspecified (null) it is set to " +
```

Comment:
This text should be updated for `neo4j-admin`. I am thinking in particular about 

> If left as unspecified (null) it is set to ...

This entire sentence can be removed. The arguments handler automatically adds a suffix to the description with the default value. Null is also a value which is never relevant.